### ISSUE TITLE
[Bug] Synchronization doesn't work with high number of labels with long strings

### ISSUE THREAD

**Issue by rhaces** [aws/graph-explorer#197]
**Community Note**
* Please use a 👍 reaction to provide a +1/vote. This helps the community and maintainers prioritize this request.
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.

**Describe the bug**
PG-Gremlin sync not working when large number of labels and label names are long. When doing a sync under these conditions the process fails with a "Unable to connect to database" error. Cannot see any additional detail

 - OS: Amazon Linux 2 (Any)
 - Browser: Any
 - Graph Explorer Version: 1.4.0
 - Graph Database & Version: Neptune 1.2.0.2 (I suspect 1.2.1 might not be affected because of the summary API)

**To Reproduce**
1. Use the python snipped code below to upload a csv to s3 bucket
```
import pandas as pd
import uuid
import string
import random
import awswrangler as wr
df = pd.DataFrame(columns=['~id','~label','name'])

max_rows = 100

for x in range(0, max_rows):
    uuid_str = uuid.uuid4()
    label_str = str(uuid_str) + ''.join(random.choices(string.ascii_lowercase +
                                        string.digits, k=100))
    df = pd.concat([pd.DataFrame([[uuid_str,label_str, "Some Random name"]],columns=df.columns), df], ignore_index=True)

wr.s3.to_csv(df = df, path = 's3://some_bucket/fail_load/bulk.csv', sep=',', index = False)
```
2. Do bulk loading into an empty Neptune cluster
3. Open Graph explorer and connect to the Neptune cluster. Notice the sync process will fail

![Screenshot from 2023-09-26 16-01-03](https://github.com/aws/graph-explorer/assets/1266195/171519a1-a0b3-4b69-b638-1a3868fcdb94)

**Expected behavior**
Correct synchronization of the connection

**Comment by abhishekpradeepmishra** [comment]
This seems to be a blocker, should be prioritized

**Comment by BrandonUSapp** [comment]
this is a big block for semantic knowledge graphs using RDF.

**Comment by xiazcy** [comment]
Issue resolved with #206.
